corehq/apps/app_manager/suite_xml.py

```diff
@@ -1913,6 +1914,44 @@ def get_target_module(case_type, module_id, with_product_details=False):
             except IndexError:
                 pass
 
+        self.add_parent_datums(datums, module)
+
+        return datums, assertions
+
+    def add_parent_datums(self, datums, module):
```

Comment:
I thought *datum* was the singular form of *data* ([wikipedia agrees](https://en.wikipedia.org/wiki/Datum)). Or maybe these are [geodetic datums](http://oceanservice.noaa.gov/facts/datum.html)? Obviously no action required.